.golangci.yml

```diff
@@ -0,0 +1,60 @@
+# Config file for golangci-lint
+run:
+  concurrency: 4
+  deadline: 1m
+  issues-exit-code: 1
+  tests: true
+  skip-dirs:
+    - .github
+    - doc
+    - docker
+    - logos
+    - scripts
+    - util
+  modules-download-mode: readonly
+
+output:
+  format: colored-line-number
+  print-issued-lines: true
+  print-linter-name: true
+
+linters:
+  disable-all: true
+  enable:
+    # - errcheck
+    # - gofmt
```

Comment:
Why disabled?

Reply:
We should run gofmt.

Reply:
I should have added a comment here — it's mostly because of a bug in `golangci-lint` on macOS Ventura where it fails unless you `brew install diffutils`.

The issue is fixed upstream but there's no tagged release that includes it yet. I guess we could use the `go install` install method to pin a more recent commit of `golangci-lint` and then enable `gofmt`? (I just didn't want to break anyone!)

Reply:
Yes that might be good, ci should have go fmt.

Or, use old method for now until it lands proper here?